conf/content_host.yaml.template

```diff
@@ -1,4 +1,6 @@
 content_host:
+  attributes:
```

Comment:
Why such nesting ? Do we plan another attributes already ?
Check supportability as a deterrent example of unnecessary nesting 
 
if I can `settings.capsule.network_type` I'd like to have also `settings.content_host.network_type`

Reply:
Resolved in f6d046cea517254d299b725ce18486a260a66649. PTAL